### ISSUE TITLE
Fasttext incorrect shape in test

### ISSUE THREAD

**Issue by menshikh-iv** [RaRe-Technologies/gensim#1370]
I run all tests and get this error (any python version, `develop` branch)
CC @prakhar2b @jayantj 
```
======================================================================
FAIL: testTraining (gensim.test.test_fasttext_wrapper.TestFastText)
Test self.test_model successfully trained, parameters and weights correctly loaded
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ivan/release/test/gensim/gensim/test/test_fasttext_wrapper.py", line 63, in testTraining
    self.assertEqual(trained_model.wv.syn0.shape, (vocab_size, model_size))
AssertionError: Tuples differ: (1763, 10) != (1762, 10)

First differing element 0:
1763
1762

- (1763, 10)
?     ^

+ (1762, 10)
?     ^


----------------------------------------------------------------------
```

**Comment by prakhar2b** [comment]
@menshikh-iv yes, we need a test for `training` new fastText model too. In PR 1319, only  test for `loading` new format was required. I'll write the code asap.

**Comment by menshikh-iv** [comment]
Fixed in [#1371](https://github.com/RaRe-Technologies/gensim/pull/1371)
